packages/react-vapor/src/components/drop/DropPod.tsx

```diff
@@ -140,9 +140,9 @@ class RDropPod extends React.PureComponent<IRDropPodProps, IDropPodState> {
                 (this.props.buttonRef.current && this.props.buttonRef.current.getBoundingClientRect()) ||
                 this.state.offset;
             const dropOffset: ClientRect | DOMRect = this.dropRef.current.getBoundingClientRect();
-            const relativeParent =
-                this.props.buttonRef?.current.closest(this.props.parentSelector) ??
-                this.props.buttonRef?.current.parentElement;
+            const relativeParent = this.props.parentSelector
+                ? this.props.buttonRef.current.closest(this.props.parentSelector)
+                : this.props.buttonRef.current.offsetParent;

```

Comment:
```
            const relativeParent =
                this.props.buttonRef?.current.closest(this.props.parentSelector) ??
                this.props.buttonRef?.current.offsetParent;
```
Is this could work for your changes without breaking the tooltip of the chart component ?

Reply:
Maybe something like 
```
const relativeParent = this.props.parentSelector
                ? this.props.buttonRef.current.closest(this.props.parentSelector) ?? this.props.buttonRef.current.offsetParent
                : this.props.buttonRef.current.offsetParent;
```
as @dmgauthier said it currently breaks the tooltip in the chart page